advanced/am-pattern-4/templates/NOTES.txt

```diff
@@ -0,0 +1,48 @@
+Thank you for installing WSO2 API Manager.
+
+Please follow these steps to access API Manager Publisher, DevPortal consoles and Analytics Dashboard.
+
+1. Obtain the external IP (`EXTERNAL-IP`) of the API Manager Ingress resources, by listing down the Kubernetes Ingresses.
+
+  kubectl get ing -n {{ .Release.Namespace }}
+
+  The output under the relevant column stands for the following.
+
+  API Manager Publisher-DevPortal
+
+  - NAME: Metadata name of the Kubernetes Ingress resource (defaults to {{ template "am-pattern-4.resource.prefix" . }}-am-ingress)
+  - HOSTS: Hostname of the WSO2 API Manager service ({{ .Values.wso2.deployment.am.pubDevPortalTM.hostname }})
+  - ADDRESS: External IP (`EXTERNAL-IP`) exposing the API Manager service to outside of the Kubernetes environment
+  - PORTS: Externally exposed service ports of the API Manager service
+
+  API Manager Gateway
+
+  - NAME: Metadata name of the Kubernetes Ingress resource (defaults to {{ template "am-pattern-4.resource.prefix" . }}-am-gateway-external-ingress and {{ template "am-pattern-4.resource.prefix" . }}-am-gateway-internal-ingress)
+  - HOSTS: Hostname of the WSO2 API Manager's Gateway service ({{ .Values.wso2.deployment.am.gateway.external.hostname }} and {{ .Values.wso2.deployment.am.gateway.internal.hostname }})
+  - ADDRESS: External IP (`EXTERNAL-IP`) exposing the API Manager's Gateway service to outside of the Kubernetes environment
```

Comment:
@chathuraswso2 has to be `External IP (`EXTERNAL-IP`) exposing the API Manager's Gateway services...`

Reply:
@chathuraswso2 please perform the same update in the README documentation.

Reply:
Fixed in https://github.com/wso2/kubernetes-apim/pull/444/commits/678355f55e4758d7c45ac915b95fe88881616bc5